### ISSUE TITLE
Improve embanked path rendering

### ISSUE THREAD

**Issue by RudyTheDev** [openstreetmap/iD#5577]
`highway=path` rendering gets really weird when they have `embankment=yes` and then `surface=unpaved`:

![p](https://user-images.githubusercontent.com/3857299/49691048-eac51d00-fb42-11e8-9b25-5effb8af5dbb.png)

From the top:
* `highway=path`
* `highway=path` , `surface=unpaved`
* `highway=path`, `embankment=yes`
* `highway=path`, `embankment=yes`, `surface=unpaved`

**Comment by bhousel** [comment]
Yeah we kind of (ab)use the line stroke for several things.  We should really draw a separate stroke for lines with a structure on them (bridge, embankment)

**Comment by matkoniecz** [comment]
<img width="792" height="51" alt="Image" src="https://github.com/user-attachments/assets/59b1b59b-1815-4602-856b-5fda32782630" />

looks fixed or at least having different problems?

now surface seems to have no effect on path styling, which may be regression of some kind

**Comment by RudyTheDev** [comment]
Oh wow, I completely forgot I even made this issue.

Yeah, the #12396 fixes this.

> now surface seems to have no effect on path styling, which may be regression of some kind

Wait, did surface ever have an (intended) effect on `path`/`footway` styling? I don't recall ever seeing anything for paths.

**Comment by matkoniecz** [comment]
per initial picture it did in 2018

**Comment by RudyTheDev** [comment]
What do you mean? The top two `highway=path` and `highway=path` + `surface=unpaved` are the same in the picture. Are you saying embankment styling was intended to represent surface values?

**Comment by matkoniecz** [comment]
as I understood from https://github.com/openstreetmap/iD/issues/5577#issue-388962381 `surface=unpaved` used to change rendering

maybe I misread something?

**Comment by RudyTheDev** [comment]
I think it only changed it for embankments and I believe that was an edge case bug and not intended. That is, paths have never had surface-based style differences.

**Comment by matkoniecz** [comment]
looking at it again, I was mistaken about `surface` showing rendering change
